specs/IR-0005/suite/test_spec_ir0005.py

```
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""IR-0005 套件——PM 优先范式（ADR-0085）条款与验收文档的结构+语义锚断言。
```

Ruff (0.16.2)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)